presto-docs/src/main/sphinx/release/release-0.229.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.229
+=============
+
+General Changes
+_______________
+* Fix an issue that would cause query failure when calling :func:`geometry_to_bing_tile` on certain degenerate geometries.
+* Add geospatial function :func:`line_interpolate_point`.
+* Add support for ``CREATE FUNCTION``.
+* Add support for using configuration property ``stage.max-tasks-per-stage`` to limit the number of tasks for scan.
+* Add support for passing ``X_Forwarded_For`` header from Proxy to coordinator.
+* Add configuration to use common ``HTTPS`` settings for internal communications.
+* Add configuration property ``experimental.internal-communication.max-task-update-size`` to limit the size of the ``TaskUpdate``.
+* Add peak total memory distribution among tasks of each stage.
```

Comment:
@viczhang861 It's unclear to me what this means. Where are we adding the peak total memory distribution? Are we adding it to the query stats?

Reply:
Also, what does "among tasks" means? Are we adding the distribution for each task or for all tasks?

Reply:
Let's change it to "Add peak task memory distribution for each stage to QueryStatistics."